app-engine/java/testSrc/com/google/cloud/tools/intellij/appengine/java/cloud/AppEngineDeploymentConfiguratorTest.java

```diff
@@ -99,4 +114,43 @@ public void createEditor_withStandardEnvironment_doesReturnStandardEditor() {
 
     assertThat(editor).isInstanceOf(AppEngineStandardDeploymentEditor.class);
   }
+
+  @Test
+  public void getDeploymentSources_withCommunityEdition_returnsSource() {
+    System.setProperty(PlatformUtils.PLATFORM_PREFIX_KEY, PlatformUtils.IDEA_CE_PREFIX);
```

Comment:
See my comment for `TestScopedSystemPropertyRule`, we could potentially move this to the rule itself and do it only once for the rule initialization call, if all test cases from this test class work with CE?

Reply:
My original intent with the `TestScopedSystemPropertyRule` was to be a generic system property setter, not specific to IDEA platform values. Are you ok with that, or would you prefer I make it specific to this use case?

Reply:
I'm ok with this now, but also, in case we start adding more features for a specific IJ runtime later, we can expand this into a new rule maybe that allows us to cleanly specify a test set will execute under specific IDE environment. wdyt?

Reply:
Oh I see your intention. Yes, that would be nice. We can definitely do that once more use-cases creep in.